components/ProjectDetails.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function ProjectDetails({ currentProject }) {
+  const { title, description, materials, duration, complexity, steps, image } =
+    currentProject;
+
+  return (
+    <>
+      <StyledProjectContainer>
+        <StyledHeadline2>{title}</StyledHeadline2>
+
+        <StyledProjectDetailsContainer>
+          <p>Duration: {duration}</p>
+          <p>Complexity: {complexity}</p>
+        </StyledProjectDetailsContainer>
+
+        <Image
+          src={image}
+          width={320}
+          height={213}
+          alt="dummy image with nails and hammer"
+        />
+
+        <p>
+          Description:
+          <br /> {description}
+        </p>
+        <ul>
+          Materials:
+          <br />
+          {materials.map((material, index) => (
+            <li key={index}>{material}</li>
+          ))}
+        </ul>
+        <ol>
+          Steps:
+          <br />
+          {steps.map((step) => (
+            <li key={step.id}>{step.desc}</li>
+          ))}
+        </ol>
+      </StyledProjectContainer>
+    </>
```

Comment:
```suggestion
      <StyledProjectContainer>
        <StyledHeadline2>{title}</StyledHeadline2>
        <StyledProjectDetailsContainer>
          <p>Duration: {duration}</p>
          <p>Complexity: {complexity}</p>
        </StyledProjectDetailsContainer>
        <Image
          src={image}
          width={320}
          height={213}
          alt="dummy image with nails and hammer"
        />
        <p>
          Description:
          <br /> {description}
        </p>
        <p>Materials:</p>
        <ul>
          {materials.map((material, index) => (
            <li key={index}>{material}</li>
          ))}
        </ul>
        <p>Steps:</p>
        <ol>
        {steps.map((step) => (
            <li key={step.id}>{step.desc}</li>
          ))}
        </ol>
      </StyledProjectContainer>
```

Reply:
The "Materials:" and the "Steps:" are not part of the list, so they should be an individual `<p>` tag :)